sktime/transformations/series/holiday/_holidayfeats.py

```diff
@@ -46,6 +47,8 @@ class HolidayFeatures(BaseTransformer):
         point is a holiday or not.
     keep_original_columns : bool, default=False
         Keep original columns in X passed to ``.transform()``.
+    raise_warnings : bool, default=False
```

Comment:
there is a `sktime` specific way to raise warnings: from `sktime.utils.warnings import warn`. Then warning scan be turned off via single interface point `set_config`.

Reply:
yeah will use this approach.